job-service-acceptance-tests/src/test/java/com/hpe/caf/jobservice/acceptance/JobServiceEndToEndIT.java

```diff
@@ -350,13 +352,15 @@ public void testJobWithPrerequisiteJobsWhichHaveCompleted() throws Exception
             createJob(job2Id, true);
         }
 
-        Thread.sleep(1000); // Add short delay to allow previous jobs to complete
+        //Wait for the job to complete
```

Comment:
Should `waitUntilJobCompletes(job1Id);` be in this test also?

Reply:
Added waitUntilJobCompletes() method for job1Id